Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -1051,6 +1051,20 @@ internal extension Purchases {
         return self.storeKit1Wrapper != nil
     }
 
+    #if DEBUG
+
+    /// - Returns: the parsed `AppleReceipt`
+    ///
+    /// - Warning: this is only meant for integration tests, as a way to debug purchase failures.
```

Comment:
❤️ 

I think this would be a useful thing to have in general with `#if debug` and just post it to debug logs. 
I bet it would save us a ton of work debugging customer issues

Reply:
we could have it print whenever a transaction goes through

Reply:
I thought about that. Are you not worried about performance implications of forcing a receipt refresh and parsing every time?

Reply:
Not too worried about it for debug builds. 
The refresh part we wouldn't really need to do, we'd just parse whatever we're about to send the backend. 
And the parsing is pretty fast in my experience. Haven't tested it again in a while, but the parser goes through the receipt only once, and leverages array slices to ensure we're not copying chunks of the receipt to different places to memory. 

The loading of the file from disk is probably the heaviest bit, and that's already done once, we could reuse it.

Reply:
[CSDK-478]

Reply:
Done in #1940. Instead of `#if DEBUG` I've checked `if Logger.logLevel == .debug` so that it can always be enabled if needed.